samples/snippets/csharp/VS_Snippets_CLR/conceptual.asyncdisposable/dispose-and-disposeasync.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
```

Comment:
Consider moving this file under the article directory.

Reply:
I did consider it, and decided to leave it for now.